app/components/new-project.js

```diff
@@ -4,10 +4,11 @@ import { action } from '@ember-decorators/object';
 
 export default class NewProjectComponent extends Component {
   @service store;
+
   @service router;
 
   constructor() {
-    super(...arguments);
+    super(...arguments); // eslint-disable-line
```

Comment:
Our ESLint rules ask us to use the rest operator in the method signature so that they're namespaced:

```
constructor(...args) {
  super(...args);
}
```